autoload/scheme.vim

```diff
@@ -1,6 +1,11 @@
 function! scheme#connect()
   new
-  let s:repl_term_id = termopen('mit-scheme')
+
+  if !exists('g:scheme_executive')
+    let g:scheme_executive = "mit-scheme"
```

Comment:
Ah, by `executive`, do you mean `executable`? I think that's the common name for this sort of thing?